import-export-cli/cmd/mg/deleteAPI.go

```diff
@@ -0,0 +1,95 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package mg
+
+import (
+	"encoding/base64"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	mgImpl "github.com/wso2/product-apim-tooling/import-export-cli/impl/mg"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+var (
+	deleteApisCmdAPIName    string
+	deleteApisCmdAPIVersion string
+	deleteApisCmdAPIVHost   string
+	deleteApisUsername      string
+)
+
+const deleteApisCmdLiteral = "apis"
```

Comment:
Does this command support deleting multiple APIs at once? If not please change this to "api" and please modify the variable names to **deleteApiCmdAPIName** like wise

Reply:
No, it only deletes one at a time. I added "apis" thinking about the endpoint. I agree that it is confusing. Updating "apis" to "api", and other var names accordingly. Thanks for noticing.